### PR TITLE
Spinners - alternate color in Style Guide

### PR DESCRIPTION
**Pull request by coliff**
Fix for: https://modus.trimble.com/components/spinners

It also fixes duplicate content issue (style and usage was the same)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## How Has This Been Tested?

locally with Edge v108 on Windows 10 Pro.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Existing tests pass locally with my changes

### CODE REVIEW

File: content/components/web/spinners/_index.md [hunk 1]
```diff
@@ -29,35 +37,3 @@ Spinners are used as indeterminate progress indicators to show the user that the
 
 - A loading scenario needs to indicate progress, usually with particularly long load times. Use a dynamically updating [Progress Bar](/components/web/progress-bars/) instead.
 - A main loading state is needed, such as an app initialization. In this case, forego our simple spinner, and opt to use a custom, branded one instead.
-
-## Specifications
```
Comment (msankaran0712) [inline content/components/web/spinners/_index.md:33]: why is this removed?

Comment (coliff) [reply content/components/web/spinners/_index.md:33]: it was duplicated content - it's on the Style page (sorry I should have mentioned in the PR above)

**Review by msankaran0712 — APPROVED** [review]
